src/bitdrift_public/fbs/issue-reporting/v1/report.fbs

```diff
@@ -0,0 +1,300 @@
+// api - bitdrift's client/server API definitions
+// Copyright Bitdrift, Inc. All rights reserved.
+//
+// Use of this source code and APIs are governed by a source available license that can be found in
+// the LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+// Specification for the format of reporting programmatic issues
+namespace bitdrift_public.fbs.issue_reporting.v1;
+
+enum ReportType : int8 {
+    Unknown = 0,
+    AppNotResponding,
+    HandledError,
+    JVMCrash,
+    MemoryTermination,
+    NativeCrash,
+    StrictModeViolation,
+}
+
+enum Platform : int8 {
+    Unknown = 0,
+    Android,
+    iOS,
+    macOS,
+}
+
+enum Architecture : int8 {
+    Unknown = 0,
+    arm32,
+    arm64,
+    x86,
+    x86_64,
+}
+
+enum FrameType : int8 {
+    Unknown = 0,
+
+    // An exception from a Java or Dalvik virtual machine
+    JVM,
+
+    // An event from C/C++ (etc) code which can be symbolicated using DWARF
+    // format files
+    DWARF,
+
+    // An event from C/C++ (etc) code on Android
+    AndroidNative,
```

Comment:
What is the distinction between these two? Can't we have a DWARF format file for android .so files?